terraform/services/api-rds/data.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   stdenv = (
-    var.app == "bcda" ? (var.env == "sbx" ? "opensbx" : var.env) :
-    var.app == "dpc" ? (var.env == "sbx" ? "prod-sbx" : var.env) :
+    var.app == "bcda" ? (var.env == "sandbox" ? "sandbox" : var.env) :
+    var.app == "dpc" ? (var.env == "sandbox" ? "prod-sandbox" : var.env) :
```

Comment:
This change would cause databases in legacy to be deleted and created under the new name. Any naming changes will need a conditional to only apply to greenfield.